components/InputField/InputField.tsx

```diff
@@ -1,33 +1,36 @@
-import React from 'react';
-import { View, TextInput, Text } from 'react-native';
+import React, { forwardRef } from 'react';
+import { View, TextInput, Text, TextInputProps } from 'react-native';
 import { globalStyles } from '../../assets/global/globalStyles';
 
 import styles from './styles';
 
-interface InputFieldProps {
+interface InputFieldProps extends TextInputProps {
   label: string;
-  placeholder: string;
+  placeholder?: string;
   value?: string;
   onChangeText?: (text: string) => void;
+  onSubmitEditing?: () => void;
 }
 
-const InputField: React.FC<InputFieldProps> = ({
-  label,
-  placeholder,
-  value,
-  onChangeText,
-}) => {
+const InputField = forwardRef<TextInput, InputFieldProps>(function InputField(
+  props,
+  ref
+) {
+  const { label, placeholder, value, onChangeText, onSubmitEditing, ...rest } =
+    props;
   return (
     <View style={styles.section}>
-      <Text style={[styles.topText, globalStyles.text]}>{label}</Text>
+      <Text style={[styles.label, globalStyles.text]}>{label}</Text>
       <TextInput
         placeholder={placeholder}
         style={[styles.input, globalStyles.text]}
         value={value}
         onChangeText={onChangeText}
+        onSubmitEditing={onSubmitEditing}
```

Comment:
Do we need `forwaredRef` for this to work?

Reply:
forwardRef is necessary when you want to pass a ref down to a child component that is not directly accessible due to being encapsulated within another component. It allows the child component to expose its own ref to the parent component. Without forwardRef, the ref you pass to the child would reference the instance of the child component itself, not the underlying DOM element or native control you're trying to access.